chpl/chpl-api/e2e/collections/search-controller.postman_collection.json

```diff
@@ -9,7 +9,7 @@
 				"listen": "test",
 				"script": {
 					"exec": [
-						"pm.test(\"/search/beta end point should return 200 status and response should have Direct Review fields when Direct Reviews are available\", function () {\r",
+						"pm.test(\"/search/beta end point should return 200 status and response should have Direct Review fields when Direct Reviews are available otherwise response status should be 400\", function () {\r",
```

Comment:
/search/beta end point should return 200 status with Direct Review fields if connected JIRA is running else response status should be 400

Something like this

Reply:
Direct Review fields are available only when JIRA is connected and the test name specifies that . But if you have a strong opinion about changing the test name , I can work on updating it.